NEWS.md

```diff
@@ -37,28 +28,20 @@ Language changes
   [IJulia](https://github.com/JuliaLang/IJulia.jl).
   Note that this only affects expressions interactively typed or pasted directly into the
   default REPL ([#28789], [#33864]).
-
 * Outside of the REPL (e.g. in a file), assigning to a variable within a top-level scope
   block is considered ambiguous if a global variable with the same name exists.
   A warning is given if that happens, to alert you that the code will work differently
   than in the REPL.
   A new command line option `--warn-scope` controls this warning ([#33864]).
-
 * Converting arbitrary tuples to `NTuple`, e.g. `convert(NTuple, (1, ""))` now gives an error,
   where it used to be incorrectly allowed. This is because `NTuple` refers only to homogeneous
   tuples (this meaning has not changed) ([#34272]).
-
-* In docstrings, a level-1 markdown header "Extended help" is now
-  interpreted as a marker dividing "brief help" from "extended help."
-  The REPL help mode only shows the brief help (the content before the
-  "Extended help" header) by default; prepend the expression with '?'
-  (in addition to the one that enters the help mode) to see the full
-  docstring. ([#25930])
-
+* In docstrings, a level-1 markdown header "Extended help" is now interpreted as a marker
```

Comment:
Is this really a language change?

Reply:
I don't know, it's kind of ambiguous since docstrings are more of a language feature. I guess it should go under `Markdown` in the stdlib section.